mono/utils/mono-error-internals.h

```diff
@@ -241,6 +241,9 @@ mono_error_set_null_reference (MonoError *error)
 	mono_error_set_generic_error (error, "System", "NullReferenceException", "");
 }
 
+// FIXME static inline failed to link for Windows
+#define mono_error_set_duplicate_wait_object(error) (mono_error_set_generic_error (error, "System", "DuplicateWaitObjectException", "Duplicate objects in argument."))
```

Comment:
Have that function like the rest of the other functions: the declaration in this file and the implementation in `mono-error.c`

Reply:
I'll try that. The link error was surprising.

Reply:
Problem was missing the include. I not used to the C behavior of allowing to call unprototyped functions and error not until link time but I'll remember it now. At least on Windows -- maybe there is a warning we can/should elevate for that, like the other compilers.